modular_tannhauser/modules/CitOwOChems/code/obj/plushies.dm

```diff
@@ -113,17 +92,7 @@ GLOBAL_LIST_INIT(valid_plushie_paths, valid_plushie_paths())
 //	if(!victim)
 //		return
 	visible_message("<span class='warning'>[src] gruesomely mutilliates [victim], leaving nothing more than dust!</span>")
```

Comment:
ummm wish i noticed this the other day. if its shreds now and not dust this will need to be changed for consistency's sake